### DISCUSSION TITLE
No-telemetry option not honored

### DISCUSSION THREAD

**Discussion by stdedos** [netdata/netdata#17790] [Q&A]
I have set BOTH `DISABLE_TELEMETRY=1` AND `.opt-out-from-anonymous-statistics`.

... but still:

![image](https://github.com/netdata/netdata/assets/133706/649ef32c-edff-4049-b8d6-f5927faa295e)

I don't know. That doesn't look like "opt-out". I'd check `/netdata.conf`, but there doesn't seem to be an option set there.

And the POSTs look pretty juicy IMHO: `eyJ1dWlkIjoiMDE4ZmNiNzItODU...`

**Comment by brodonalds** [comment]
This was after you created the empty opt out file and installed with the --disable-telemetry flag?

**Reply by stdedos** [reply]
Yes. The helm app has been edited since (I am seeing a different hostname, which was configured definitely after the application started)

**Reply by brodonalds** [reply]
Helm appears to be a bit different for opt out. https://learn.netdata.cloud/docs/netdata-agent/installation/kubernetes-helm-chart-reference#service-discovery-and-supported-services

Just above the configuration files header is the below. Hope it helps.

Note:: To opt out of anonymous statistics, set the DO_NOT_TRACK environment variable to non-zero or non-empty value in parent.env / child.env configuration (e.g: DO_NOT_TRACK: 1) or uncomment the line in values.yml.

**Reply by stdedos** [reply]
Kindly:

1. Update your opt-out docs
2. "Somehow" make that option accessible via a "similarly-convenient" endpoint like https://netdata/netdata.conf

I don't see why do you need to use yet-another name for `DO_NOT_TRACK`. I'd tell you to make them the same, but I guess that won't happen.

**Reply by stdedos** [reply]
_Also_, still happening

![image](https://github.com/netdata/netdata/assets/133706/648b785d-ab7e-4361-9c7d-d07b7636730d)

"I think" I did everything right, but I am a bit way over my head here (using https://truecharts.org/charts/stable/netdata/)

**Reply by brodonalds** [reply]
Have you tried using their official installation guide for helm vs the truecharts one? I wonder if truecharts ignores some of the install flags or doesn't support them.

**Reply by stdedos** [reply]
Well, that's a bit of a troubling thing to do - as _helm_ verbatim is not available in TrueNAS

**Reply by stdedos** [reply]
So? Any progress??
